### PR TITLE
Support for renaming ign to gz names

### PR DESCRIPTION
**Pull request by j-rivero**
Add duplicate entries for ignition packages to support the rename of them to gazebo.

### CODE REVIEW

File: plugins/config/repository.yaml [hunk 1]
```diff
@@ -24,6 +24,116 @@ repositories:
 # wildcards are allowed in name, entries are processed in top-down order
 # first entry matching the name is used
 projects:
+    # Use nightlies in all main branches
+    - name: gazebo-cmake3
```
Comment (chapulina) [inline plugins/config/repository.yaml:28]: These should all be `gz`

File: plugins/config/repository.yaml [hunk 2]
```diff
@@ -24,6 +24,116 @@ repositories:
...
+            type: stable
+          - name: osrf
+            type: nightly
+    - name: sdformat13
+      repositories:
+          - name: osrf
+            type: stable
+          - name: osrf
+            type: nightly
+    - name: gazebo-gazebo6
```
Comment (chapulina) [inline plugins/config/repository.yaml:130]: This one is being removed in https://github.com/ignition-tooling/gzdev/pull/58

File: plugins/config/repository.yaml [hunk 3]
```diff
@@ -24,6 +24,116 @@ repositories:
...
+            type: stable
+          - name: osrf
+            type: nightly
+    - name: gazebo-garden
+      repositories:
+          - name: osrf
+            type: stable
+          - name: osrf
+            type: nightly
+    - name: gazebo-gazebo7
```
Comment (chapulina) [inline plugins/config/repository.yaml:52]:
```suggestion
    - name: gz-sim7
```

**Review by chapulina — CHANGES_REQUESTED** [review]

File: plugins/config/repository.yaml (hunk 2 above)
Comment (j-rivero) [reply plugins/config/repository.yaml]: [93406ea](https://github.com/ignition-tooling/gzdev/pull/57/commits/93406ea1b035e75dd8912d7d3b048708058254f5)

File: plugins/config/repository.yaml (hunk 1 above)
Comment (j-rivero) [reply plugins/config/repository.yaml]: [6189aee](https://github.com/ignition-tooling/gzdev/pull/57/commits/6189aee4ea36daee7b259078a68f80f65142783c)

File: plugins/config/repository.yaml (hunk 3 above)
Comment (j-rivero) [reply plugins/config/repository.yaml]: [9478706](https://github.com/ignition-tooling/gzdev/pull/57/commits/9478706fab388e820b6f64bd51df0d6c453f043e)

**Review by chapulina — APPROVED** [review]

**Pull request merged**